src/main/java/io/jenkins/plugins/audit/helpers/Factory.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.audit.helpers;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class Factory {
+    /**
+     * Returns a date string in the ISO-8601 format
+     * @param milliseconds Time in epoch milliseconds to be formatted
+     */
+    public static String formatDateISO(Long milliseconds) {
```

Comment:
```suggestion
    public static String formatDateISO(long milliseconds) {
```

We would only want to use `Long` if it's possible for the value to be `null`, and since we don't have a real return value for `null` here, it makes more sense to force it as a primitive value here.